pkg/collector/py/datadog_agent.c

```diff
@@ -101,12 +110,115 @@ static PyObject *get_subprocess_output(PyObject *self, PyObject *args) {
     return py_result;
 }
 
+static PyObject *add_external_tags(PyObject *self, PyObject *args) {
+    char *hostname, *source_type;
+    int input_len;
+    PyObject *input_list, *tuple, *dict, *key, *value;
```

Comment:
Not a must have, but C doesn't initialize to zero-values (like Go). Suggest:
`PyObject *input_list = NULL, *typle = NULL, *dict = NULL, *key = NULL, *value = NULL`